tests/test_globals.hpp

```diff
@@ -60,7 +58,12 @@ class Global
 #endif
 
 	Global() {
-		wnd.create(sf::VideoMode(640, 360), "Testing");
+		if (SDL_Init(SDL_INIT_VIDEO) < 0)
+			throw std::runtime_error("Failed to initialize SDL2!");
+
+		window.create(800, 600, false);
```

Comment:
nit: I think test mode should always run at 640x480 because that's what GTA3 seems to use as it's reference coordinate system?
(At least we should keep the intended 4:3 AR)